dwave/gate/mixedproperty.py

```diff
@@ -12,6 +12,11 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
+""":class:`mixedproperty` decorator.
+
+Contains a decorator for creating mixed properties, which differs from regular properties by
```

Comment:
"creating mixed properties, which differs" --> the singular "differs" will take "decorator" for its subject but you mean the plural "properties".